.travis.yml

```diff
@@ -83,7 +83,7 @@ matrix:
       script: bundle exec rake chef_zero_spec
       env:
         - "PEDANT_OPTS=\"--skip-oc_id\""
-        - "GEMFILE_MOD=\"gem 'rake'; gem 'chef-zero', github: 'chef/chef-zero', tag: 'v14.0.5' \""
+        - "GEMFILE_MOD=\"gem 'rake'; gem 'chef-zero', github: 'chef/chef-zero'; gem 'chef', github: 'chef/chef' \""
```

Comment:
The pin to 14.0.5 was due to the latest chef-zero having issues with chef 13; it might be better to wait on this unpinning until we no longer pin the rest of chef server to chef 13.

Reply:
it actually does no good anyway, so yeah, i'll revert it.